README.md

```diff
@@ -71,21 +85,21 @@ requirements refer to [pv_cheetah.h](/include/pv_cheetah.h). The following trans
 resource directory.
 
 ```bash
-python demo/python/cheetah_demo.py --audio_paths resources/audio_samples/test.wav
+python demo/python/cheetah_demo.py --audio_paths resources/audio_samples/test.wav --license_path PATH_TO_YOUR_CHEETAH_LICENSE_FILE
 ```
 
 In order to transcribe multiple files concatenate their paths using comma as below.
 
 ```bash
-python demo/python/cheetah_demo.py --audio_paths PATH_TO_AUDIO_FILE_1,PATH_TO_AUDIO_FILE_2,PATH_TO_AUDIO_FILE_3
+python demo/python/cheetah_demo.py --audio_paths PATH_TO_AUDIO_FILE_1,PATH_TO_AUDIO_FILE_2,PATH_TO_AUDIO_FILE_3 --license_path PATH_TO_YOUR_CHEETAH_LICENSE_FILE
 ```
 
 ### Realtime Demo
 
 This Cheetah demo records audio from the microphone and transcribes it in real-time:
 
 ```bash
-python demo/python/cheetah_demo_realtime.py
+python demo/python/cheetah_demo_realtime.py --license_path PATH_TO_YOUR_CHEETAH_LICENSE_FILE
```

Comment:
We used ${PATH_TO_YOUR_CHEETAH_LICENSE_FILE} notation for variables in shell across documentation.